templates/hackernews/handlers/story.ts

```
@@ -0,0 +1,61 @@
+        return {
+            buttons: [
+                { label: 'Visit', action: 'link', target: story.url },
+                { label: 'Next' },
+            ],
+            fonts,
+            component: StoryView(story),
+            handler: 'story',
+            storage: { ...storage, viewedStories },
+            params: { currentIndex },
+        }
+    }
```

Biome

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)